core/src/test/scala/org/apache/spark/storage/FallbackStorageSuite.scala

```diff
@@ -227,43 +228,45 @@ class FallbackStorageSuite extends SparkFunSuite with LocalSparkContext {
     }
   }
 
-  test("Newly added executors should access old data from remote storage") {
-    sc = new SparkContext(getSparkConf(2, 0))
-    withSpark(sc) { sc =>
-      TestUtils.waitUntilExecutorsUp(sc, 2, 60000)
-      val rdd1 = sc.parallelize(1 to 10, 2)
-      val rdd2 = rdd1.map(x => (x % 2, 1))
-      val rdd3 = rdd2.reduceByKey(_ + _)
-      assert(rdd3.collect() === Array((0, 5), (1, 5)))
+  Seq("lz4", "snappy", "zstd").foreach { codec =>
```

Comment:
The doc of `IO_COMPRESSION_CODEC` says that it supports `lz4, lzf, snappy, and zstd`. Should we test `lzf` too?

Reply:
Sure, I'll add that, @MaxGekk .